packages/checkout/totals/taxes/index.tsx

```diff
@@ -40,29 +40,38 @@ const TotalsTaxes = ( {
 		return null;
 	}
 
-	const itemisedTaxItems: ReactElement | null = getSetting(
+	const showItemisedTaxes = getSetting(
 		'displayItemizedTaxes',
 		false
-	) ? (
-		<>
-			{ taxLines.map( ( { name, rate }, i ) => {
+	) as boolean;
+
+	const itemisedTaxItems: ReactElement | null = showItemisedTaxes ? (
+		<div
+			className={ classnames(
+				'wc-block-components-totals-taxes',
+				className
+			) }
+		>
+			{ taxLines.map( ( { name, rate, price }, i ) => {
 				const label = `${ name }${
 					showRateAfterTaxName ? ` ${ rate }` : ''
 				}`;
 				return (
 					<TotalsItem
 						key={ `tax-line-${ i }` }
-						className="wc-block-components-totals-taxes__tax-line"
```

Comment:
Any reason we changed this classname? was it introduced recently in your PR or was it there since before?

Reply:
It's new as of the previous PR, but I changed it to be more descriptive of what the element actually represents.